packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,72 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown id="DropdownExample">Dropdown Content</Dropdown>
+```
+
+Usage with link:
+
+```twig
+<ButtonLink href="javascript:void(0)" data-toggle="dropdown" data-target="CollapseExample">Open Dropdown</ButtonLink>
+...
+```
+
+Responsive usage for tablet
+
+```twig
+...
+<Button class="d-tablet-none">Open Dropdown</Button>
+<Dropdown id="DropdownExample" breakpoint="tablet">Dropdown Content</Dropdown>
+```
+
+Advanced example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown elementType="span" id="DropdownExample" class="custom-dropdown-class">Dropdown Content</Dropdown>
+```
+
+Without lexer:
+
+```twig
+{% embed "@spirit/dropdown.twig" with { props: {
+    breakpoint: 'tablet',
+    class: 'dropdown-custom-class',
+    elementType: 'span'
+}} %}
+    {% block content %}
+          Dropdown content
+    {% endblock %}
+{% endembed %}
+```
+
+You can add any custom trigger like `<button>` or `<a>` but it is necessary to add `data-toggle="dropdown"`, `data-target="<id>"`
+attributes to register trigger events.
+
+## API
+
+| Prop name     | Type     | Default | Required | Description                              |
+| ------------- | -------- | ------- | -------- | ---------------------------------------- |
+| `id`          | `string` | -       | yes      | Dropdown ID                              |
+| `breakpoint`  | `string` | -       | no       | Breakpoint level [mobile,tablet,desktop] |
+| `class`       | `string` | `null`  | no       | Custom CSS class                         |
+| `elementType` | `string` | `div`   | no       | HTML tag to render                       |
+
+## Trigger attributes
+
+| Prop name       | Type     | Default    | Required | Description                |
+| --------------- | -------- | ---------- | -------- | -------------------------- |
+| `data-toggle`   | `string` | `dropdown` | yes      | Iterable selector          |
+| `data-target`   | `string` | -          | yes      | Target selector            |
+| `aria-expanded` | `string` | -          | no       | Aria expanded state (auto) |
+| `aria-controls` | `string` | -          | no       | Aria controls state (auto) |
```

Comment:
These two options should be present in the examples too. I know that `aria` attributes are/were injected by JS at some point, but we need them to be present in HTML even _before_ JS is initialized.

Reply:
(Side note: I don't know how yet, but it would be good to make developers' life easier by providing a `DropdownTrigger` component (or hook in React) that makes all these mandatory things automatically. I'm filing this to be discussed later.)

Reply:
> (Side note: I don't know how yet, but it would be good to make developers' life easier by providing a `DropdownTrigger` component (or hook in React) that makes all these mandatory things automatically. I'm filing this to be discussed later.)

@adamkudrna Please, file an issue for this. Thanks.

Reply:
I've put it on the Thursday's topics list. I'm afraid it's not ready for an issue yet.